recipes/goodtables/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "goodtables" %}
+{% set version = "2.1.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e363d0d2762f5ac48b5f408bfe45c0de1ce8bad908cff42464bb11616ad40e55
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - goodtables = goodtables.cli:cli
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - six>=1.9
```

Please leave a space between name and version spec for all dependencies.
```suggestion
    - six >=1.9
```